src/common/NetworkPrivate.cpp

```diff
@@ -174,7 +174,7 @@ void loadUncached(std::shared_ptr<NetworkData> &&data)
         }
 
         auto handleReply = [data, reply]() mutable {
-            if (data->hasCaller_ && !data->caller_.get())
+            if (data->hasCaller_ && !data->caller_.data())
```

Comment:
Since we are not using the returned pointer, use QPointers [isNull](https://doc.qt.io/qt-6/qpointer.html#isNull) function instead

Same goes for the rest of the data() checks in this file